### DISCUSSION TITLE
Simultaneous state events?

### DISCUSSION THREAD

**Discussion by anaeguiguren** [olivierfriard/BORIS#740] [Q&A]
Hi! I am trying to keep track of contact behavioiurs between individuals, and was hoping to annotate contact events of the same kind (e.g., behaviour = contact)  involving different individuals simmultaneously. However, when I start a new contact event before a previous one has finished, the softwaare automatically ends the previous event and starts a new one. Is there a way around this? Thanks very much for your help!

**Comment by olivierfriard** [comment]
Hi,
it is not possible with the same behavior.
You have to define different behaviors to track the simultaneous contacts.
